cli/.eslintrc

```diff
@@ -3,28 +3,18 @@
   "env": {
     "es6": true
   },
-  "plugins": [
-    "flowtype",
-    "prettier"
-  ],
-  "extends": [
-    "plugin:flowtype/recommended"
-  ],
-  "globals": {
-  },
+  "plugins": ["flowtype"],
```

Comment:
Tests are failing, probably needs to keep `"prettier"` in plugins?

Reply:
According to the docs, `plugin:prettier/recommended` does three things:

- Enables `eslint-plugin-prettier`.
- Sets the `prettier/prettier` rule to `"error"`.
- Extends the `eslint-config-prettier` configuration.

I'll have to investigate why the tests are failing

Reply:
okay apparently we did need to keep `eslint-config-prettier` in the dev deps